crypto/signing/mcl/suiteBLS12_381.go

```diff
@@ -94,21 +88,7 @@ func (s *SuiteBLS12) CreateScalar() crypto.Scalar {
 
 // CreatePointForScalar creates a new point corresponding to the given scalar
 func (s *SuiteBLS12) CreatePointForScalar(scalar crypto.Scalar) (crypto.Point, error) {
-	if check.IfNil(scalar) {
-		return nil, crypto.ErrNilPrivateKeyScalar
-	}
-	sc, ok := scalar.GetUnderlyingObj().(*bls.Fr)
-	if !ok {
-		return nil, crypto.ErrInvalidScalar
-	}
-
-	if sc.IsZero() || !sc.IsValid() {
-		return nil, crypto.ErrInvalidPrivateKey
-	}
-
-	point := s.G2.CreatePointForScalar(scalar)
-
-	return point, nil
+	return s.G2.CreatePointForScalar(scalar), nil
```

Comment:
I guess the removal of the checks was unintended

Reply:
undone